features/bevragen/zoek-met-postcode-en-huisnummer/dev/fout-cases-gba.feature

```diff
@@ -325,3 +325,31 @@ Regel: Alleen gespecificeerde parameters bij het opgegeven zoektype mogen worden
     | zoeken met parameter uit ander zoektype   | voornamen   | Pietje     |
     | typfout in naam optionele parameter       | huisleter   | A          |
     | zoeken met niet gespecificeerde parameter | bestaatNiet | een waarde |
+
+Regel: De geboortedatum is een datum string geformatteerd volgens de [ISO 8601 date format](https://www.w3.org/QA/Tips/iso-date)
```

Comment:
Deze scenario geldt ook voor de Autorisatie en Protocollering service (brp-shared-dotnet repo)

Reply:
Scenario is toegevoegd aan brp-shared-dotnet

[#41 feat(zoek-met-postcode-en-huisnummer)](https://github.com/BRP-API/brp-shared-dotnet/pull/41)